src/styles/sidebar/components/annotation-body.scss

```diff
@@ -38,12 +33,12 @@
 }
 
 .annotation-body__collapse-toggle {
-  // Negative top margin to bring this up tight under `.annotation-body`
-  margin-top: -(var.$layout-h-margin - 5px);
+  margin: 0.5em 0;
```

Comment:
This increases whitespace, but it sounds like your PR's intent is to remove some? Is this intentional for this button?

Reply:
Overall, decreasing whitespace in this PR, but the overarching goal is making the vertical rhythm consistent. The structure around this particular element was adjusted in this PR, so the negative margin hack thingy is no longer necessary (if left there, the button actually overlaps the content above it).

Reply:
okay thanks.